content/learn/beginner/datatypes-1.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Data Types 1"
+weight = 220
++++
+
+In this chapter, we will overview how to store data in variables and the basic data types available, which include
+numbers, strings, and more.
+
+## Variables
+
+Before we start, we have to find a way to store data and refer to it later. We can do this by using variables. A
+variable is a way to store a value and give it a name, to which you can refer back to later.
+
+In Custom Commands, you define a variable by starting with a `$`, its name, and the assignment operator `:=`. To
+illustrate this, consider the following example:
+
+```go
+{{ $name := "Alice" }}
+{{ $age := 42 }}
+{{ $isCool := true }}
+
+{{ $name }}, aged {{ $age }}, is cool: {{ $isCool }}
+```
+
+Later on, we may wish to reassign a new value to a variable. We can do this by using the re-assignment operator `=`.
+When and why this becomes necessary will be discussed in a later chapter.
+
+{{< callout context="tip" title="Tip" icon="outline/rocket" >}}
+
+When debugging your code, you might have to figure out the type of a certain variable. You can do this by using the
+[`printf` function](/docs/reference/templates/functions/#string-manipulation) with the `%T` verb, like so:
+
+```go
+{{ $name := "Alice" }}
+{{ printf "%T" $name }}
+```
+
+{{< /callout >}}
+
+## Data Types
+
+If you're completely new to programming, you might not know what a data type is. You can think of a data type as a way
+to distinguish between different kinds of "things". As a real-life analogy, you can separate food into several
+categories, such as fruits, vegetables, and meat. Each category is in that sense its own "data type".
```

Comment:
American English puts punctuation inside the quotation mark; I personally dislike this and follow the British convention of putting (non-source) punctuation outside the quotation. That said, in this case I'd drop the quotation marks entirely.

Reply:
As much as it pains me, too, we should stick to American English conventions simply for consistency's sake—we're already using the US spelling (e.g. color instead of colour), therefore it doesn't make much sense to suddenly introduce UK conventions on top.


(Already said that on Discord, just regurgitating here for the record)

Reply:
Lord save me from this nightmare

Do not go gentle into that good night,
Rage, rage against the dying of the light.